site/components/src/main/java/uk/nhs/digital/website/beans/UserNeed.java

```diff
@@ -0,0 +1,24 @@
+package uk.nhs.digital.website.beans;
+
+import org.hippoecm.hst.content.beans.Node;
+import org.hippoecm.hst.content.beans.standard.HippoCompound;
+import org.onehippo.cms7.essentials.dashboard.annotations.HippoEssentialsGenerated;
+
+@HippoEssentialsGenerated(internalName = "website:userNeed")
+@Node(jcrType = "website:userNeed")
+public class UserNeed extends HippoCompound {
+    @HippoEssentialsGenerated(internalName = "website:Ineedto")
```

Comment:
This should start with lowercase 'I' in `website:Ineedto` and it corresponding YMAL.

Reply:
> This should start with lowercase 'I' in `website:Ineedto` and it corresponding YMAL.

Indeed, this will be corrected.